test/e2e/scenario/agents.scenario.ts

```diff
@@ -73,7 +74,31 @@ describe('rum', () => {
   })
 
   it('should track xhr timings', async () => {
-    const timing = await makeXHRAndCollectEvent(`${browser.config.baseUrl}/ok`)
+    const timing = await makeXHRAndCollectEvent(`${serverUrl.sameOrigin}/ok`)
+    expect(timing).not.toBeUndefined()
+    expect(timing!.http.method).toEqual('GET')
+    expect((timing!.http as any).status_code).toEqual(200)
+    expectToHaveValidTimings(timing!)
+  })
+
+  it('should not track disallowed cross origin xhr timings', async () => {
+    const timing = await makeXHRAndCollectEvent(`${serverUrl.crossOrigin}/ok`)
+    expect(timing).not.toBeUndefined()
+    expect(timing!.http.method).toEqual('GET')
+    expect((timing!.http as any).status_code).toEqual(200)
+    expect(timing!.duration).toBeGreaterThan(0)
+
+    // Edge 18 seems to have valid timings even on cross browser requests ¯\_ツ_/¯ It doesn't matter
+    // too much.
+    if (browser.capabilities.browserName === 'MicrosoftEdge' && browser.capabilities.browserVersion === '18') {
```

Comment:
Chrome based Edge does not seem to be rollout by microsoft yet, this behavior could be fixed with it.

Reply:
The Chrome based Edge has been released and stable since 15 january... but you are right, it doesn't auto-update on all Windows setups... things will be better in the future :)